### ISSUE TITLE
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED]

### ISSUE THREAD

**Issue by naurasd** [DominikBuchner/BOLDigger-commandline#15]
Hi,

using version 2.2.1. I am getting a ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED]
```
$ boldigger-cline ie_coi "username" "password" COI_cluster_reps_lulu_curated.fa ~/result 50
Traceback (most recent call last):
  File "c:\users\nauras\programs\python\python39\lib\site-packages\urllib3\connectionpool.py", line 699, in urlopen
    httplib_response = self._make_request(
  File "c:\users\nauras\programs\python\python39\lib\site-packages\urllib3\connectionpool.py", line 382, in _make_request
    self._validate_conn(conn)
  File "c:\users\nauras\programs\python\python39\lib\site-packages\urllib3\connectionpool.py", line 1010, in _validate_conn
    conn.connect()
  File "c:\users\nauras\programs\python\python39\lib\site-packages\urllib3\connection.py", line 411, in connect
    self.sock = ssl_wrap_socket(
  File "c:\users\nauras\programs\python\python39\lib\site-packages\urllib3\util\ssl_.py", line 428, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "c:\users\nauras\programs\python\python39\lib\site-packages\urllib3\util\ssl_.py", line 472, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "c:\users\nauras\programs\python\python39\lib\ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "c:\users\nauras\programs\python\python39\lib\ssl.py", line 1040, in _create
    self.do_handshake()
  File "c:\users\nauras\programs\python\python39\lib\ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1122)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\nauras\programs\python\python39\lib\site-packages\requests\adapters.py", line 439, in send
    resp = conn.urlopen(
  File "c:\users\nauras\programs\python\python39\lib\site-packages\urllib3\connectionpool.py", line 783, in urlopen
    return self.urlopen(
  File "c:\users\nauras\programs\python\python39\lib\site-packages\urllib3\connectionpool.py", line 783, in urlopen
    return self.urlopen(
  File "c:\users\nauras\programs\python\python39\lib\site-packages\urllib3\connectionpool.py", line 783, in urlopen
    return self.urlopen(
  [Previous line repeated 12 more times]
  File "c:\users\nauras\programs\python\python39\lib\site-packages\urllib3\connectionpool.py", line 755, in urlopen
    retries = retries.increment(
  File "c:\users\nauras\programs\python\python39\lib\site-packages\urllib3\util\retry.py", line 573, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='boldsystems.org', port=443): Max retries exceeded with url: /index.php/Login (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1122)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\nauras\programs\python\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\nauras\programs\python\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\Nauras\Programs\Python\Python39\Scripts\boldigger-cline.exe\__main__.py", line 7, in <module>
  File "c:\users\nauras\programs\python\python39\lib\site-packages\boldigger_cline\__main__.py", line 87, in main
    boldblast_coi.main(
  File "c:\users\nauras\programs\python\python39\lib\site-packages\boldigger_cline\boldblast_coi.py", line 14, in main
    session = login(username, password, certs)
  File "c:\users\nauras\programs\python\python39\lib\site-packages\boldigger_cline\login.py", line 35, in login
    session.post("https://boldsystems.org/index.php/Login", data=data)
  File "c:\users\nauras\programs\python\python39\lib\site-packages\requests\sessions.py", line 590, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "c:\users\nauras\programs\python\python39\lib\site-packages\requests\sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "c:\users\nauras\programs\python\python39\lib\site-packages\requests\sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "c:\users\nauras\programs\python\python39\lib\site-packages\requests\adapters.py", line 514, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='boldsystems.org', port=443): Max retries exceeded with url: /index.php/Login (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1122)')))
```

Not sure what is going on here, have run things before without receiving this error message. I can login to the BOLD webserver, so this is not the problem.

Best
Nauras

**Comment by DominikBuchner** [comment]
Fixed with 2.2.3. However, please consider upgrading to BOLDigger2 :)

**Comment by naurasd** [comment]
will do ;-) thanks so much.
